platform/linux-generic/odp_ishm.c

```diff
@@ -1094,12 +1111,15 @@ int _odp_ishm_reserve(const char *name, uint64_t size, int fd,
 
 	new_block = &ishm_tbl->block[new_index];
 
-	/* save block name (if any given): */
-	if (name)
-		_odp_strcpy(new_block->name, name, ISHM_NAME_MAXLEN);
-	else
+	/* Save application given block name as-is and make internal block name used for mmap
+	   unique */
```

Comment:
The call to alloc_single_va() later on in this function still uses the application given non-unique name. I did not read carefully but it looks like it would be a problem for exported memory blocks as it would cause overlapping export file names.

Reply:
Was not a problem but logic changed in V3.